src/main/groovy/org/pih/warehouse/inventory/CycleCountItemCommand.groovy

```diff
@@ -38,6 +41,11 @@ class CycleCountItemCommand implements Validateable {
 
     static constraints = {
         quantityCounted(nullable: true)
+        discrepancyReasonCode(nullable: true, validator: { ReasonCode discrepancyReasonCode ->
+            return ReasonCode.listInventoryAdjustmentReasonCodes().contains(discrepancyReasonCode) ?
+                    true :
+                    ['cycleCountItemCommand.discrepancyReasonCode.invalid']
+        })
```

Comment:
This is just an idea, not a change request.
Maybe it will be worth having. a place where we can store our validationrs to keep the classes cleaner? I can see that this closure is duplicated.